### PR TITLE
General FAQ and Account Disabled Fix

### PR DESCRIPTION
**Pull request by Priestytheplushie**
Kinda a bad guide but a general FAQ should be added probably, feel free to add more if you want

### CODE REVIEW

**Comment by Priestytheplushie** [comment]
Why isn't the markdown working?

**Comment by nojnis** [comment]
looks fine so far

**Comment by Priestytheplushie** [comment]
Yeah but it's not complete, I need like 7 or 8 more faqs

File: faq/general-faq.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Assign a role in phpmyadmin or use the GDPSFN panel, check out [This](phpmyadmin/create-roles.md) guide for a detailed explanation of permissions
+
+### Why does most of the panel not work?
+The panel is still under development, you can accomplish bascially everything the panel does in phpnyadmin.
+
+### Can i add 2.2? 
+Not without a mod, GDPSFN does not support 2.2 at this time
+
+### I can't complete 3 star and above levels that are shorter than 30 seconds
+You need to disable anticheat
```
Comment (Jecket22) [inline faq/general-faq.md]: aha. good answer

Comment (Priestytheplushie) [reply faq/general-faq.md]: well i will link the remove anti-cheat page there

Comment (Priestytheplushie) [reply faq/general-faq.md]: Ok fixed

**Comment by Jecket22** [comment]
I think I'll create a FAQ section on my own. I'll take a few lines though, thank you anyway

**Pull request closed without merging**